pkg/storage/storage.go

```diff
@@ -111,7 +112,7 @@ func (is *ImageStore) InitRepo(name string) error {
 			is.log.Panic().Err(err).Msg("unable to marshal JSON")
 		}
 
-		if err := ioutil.WriteFile(ilPath, buf, 0644); err != nil {
+		if err := ioutil.WriteFile(ilPath, buf, 0600); err != nil {
```

Comment:
What's the reason for this change? This just makes it more annoying to introspect things as non-zot, which I expect we'll want to do at some point. Is there a threat model or something?

Reply:
After upgrading golangci-lint and go version, gosec linter highlighted this and suggested the permission should be 0600 or less, so i changed it to 0600, if a permission should be 0644 i can add nolint to it.

Reply:
I think it should be 644. Linters (especially go ones) are often not correct :)

Reply:
Ok, I will do the necessary changes.

Reply:
Well, could really be writable only on first create - 0400 would be fine also if the subsequent access is only needed for zot daemon and via HTTP.

Reply:
Ok, should I change it to 0400 ?

Reply:
No, leave it at 644 for now. We can address it in a separate PR if needed.
This is the gosec error.
G306: Poor file permissions used when writing to a new file

Reply:
> access is only needed for zot daemon

I don't think that's true, though.

Reply:
Changed back to 0644